Server/mods/deathmatch/logic/CPed.cpp

```diff
@@ -112,6 +112,7 @@ CElement* CPed::Clone(bool* bAddEntity, CResource* pResource)
         pTemp->SetHealth(GetHealth());
         pTemp->SetArmor(GetArmor());
         pTemp->SetSyncable(IsSyncable());
+        pTemp->SetSpawned(true);

```

Comment:
```suggestion
        pTemp->SetSpawned(IsSpawned());
        pTemp->SetIsDead(IsDead());
```
What happens if the ped is dead and then cloned?

I think it might make sense to use `IsSpawned` here.

You may also want to add the same for `SetIsDead` using `IsDead`

Reply:
Good suggestion. It did not work properly for dead peds also.